internal/extender/resource.go

```diff
@@ -445,9 +447,9 @@ func (s *SparkSchedulerExtender) rescheduleExecutor(ctx context.Context, executo
 	availableNodes := s.getNodes(ctx, nodeNames)
 	usages := s.usedResources()
 	usages.Add(s.overheadComputer.GetOverhead(ctx, availableNodes))
-	availableResources := resources.AvailableForNodes(availableNodes, usages)
+	availableNodesSchedulingMetadata := resources.NodeSchedulingMetadataForNodes(availableNodes, usages)
```

Comment:
I think for rescheduling you can keep the available resources, as nothing else in the metadata is used